eZ/Publish/API/Repository/ObjectStateService.php

```diff
@@ -47,6 +47,31 @@ public function createObjectStateGroup(ObjectStateGroupCreateStruct $objectState
      */
     public function loadObjectStateGroup($objectStateGroupId, array $prioritizedLanguages = []);
 
+    /**
+     * Loads an object state by identifier and group it belongs to.
+     *
+     * @param string $identifier
+     * @param mixed $groupId
+     * @param string[] $prioritizedLanguages Used as prioritized language code on translated properties of returned object.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if the state was not found
+     *
+     * @return \eZ\Publish\API\Repository\Values\ObjectState\ObjectState
+     */
+    public function loadByIdentifier($identifier, $groupId, array $prioritizedLanguages = []);
```

Comment:
not too sure about this. would it be better with group identifier ?

Reply:
I'm fine either way.  I just followed the example in the existing spi handler